code/game/atoms.dm

```diff
@@ -132,6 +132,9 @@
 /atom/proc/is_open_container()
 	return flags & OPENCONTAINER
 
+atom/proc/is_transparent()
+	return flags & TRANSPARENT
+

```

Comment:
I feel like these procs can be turned into is_helpers DEFINES, which reduces our proc count. Because apparently in DM, proc calls are expensive. At least, that's what @RemieRichards tells me